index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="css/reset.css">
+        <link rel="stylesheet" href="css/mobile-style.css">
+        <link rel="stylesheet" media="(min-width: 768px)" href="css/tablet-style.css" />
+        <link rel="stylesheet" media="(min-width: 1024px)" href="css/desktop-style.css" />
+        <title>Next Web Clone</title>
+    </head>
+    <body>
+        <nav id="navbar">
```

Comment:
- [ ] **Enclose the nav inside a `<header>` tag**
- **Remember the use of [semantics](https://www.w3schools.com/html/html5_semantic_elements.asp)**